deploy_server.sh

```diff
@@ -39,10 +40,42 @@ InitSubmodules() {
 
 StartServer() {
     cp ${SOURCE_DIR}/${TARGET_SCRIPT} ${TARGET_DIR}
-    python3 ${TARGET_DIR}/${TARGET_SCRIPT} --host 127.0.0.1 --ws-port 8081
+    python3 ${TARGET_DIR}/${TARGET_SCRIPT} ${ARGS}
     rm ${TARGET_DIR}/${TARGET_SCRIPT}
 }
 
+Help() {
+    printf "Usage: deploy_server.sh <argument> <value>\n\n"
+    echo "List of available arguments:"
+    echo "-h, --h, -help, --help     Show this help message and exit. Called without value"
+    echo "--host                     Backend server hostname. Default value: 127.0.0.1"
+    echo "--ws-port                  Backend server port number. Default value: 8081"
+    echo "--video-port               Video streaming server port number"
+    echo "--audio-port               Audio streaming server port number"
+    echo "--cert-passphrase          A secret password used to decrypt the certificate from the PT"
+    echo "--fs-port                  File server port number"
+    echo "--fs-uri                   File server's URI (to be sent back to the client hmi)"
+}
+
+if ! [ $# -eq 0 ]
+    then
+    while [ $# -gt 0 ]; do
+        if [ "$1" == "--h" ] || [ "$1" == "--help" ] || [ "$1" == "-h" ] || [ "$1" == "-help" ]  ; then
+            Help
+            exit 0
+        fi
+        ARGS="${ARGS} $1"
+        shift
+    done
+fi
+
+if [[ $ARGS != *"--host"* ]]; then
+    ARGS="${ARGS} --host 127.0.0.1"
+fi
+if [[ $ARGS != *"--ws-port"* ]]; then
+    ARGS="${ARGS} --ws-port 8081"
+fi
```

Comment:
@ValeriiMalkov I would suggest making a single responsibility for default params. It can be done in 2 ways:
1. Remove default values from that script and declare them in `Flags()` class in `start_server.py` like for other parameters
2. Remove class `Flags()` in `start_server.py` and make this script default the rest of params from there.

And update help correspondingly with information about default values for each param.
I would prefer Option 1

Reply:
@AKalinich-Luxoft There are a third way which is done here 48518eca250e7e07dda9d08d51209891a159db7e
Default values added to the `ArgumentParser`

Reply:
@ValeriiMalkov cool hommie... but where is the help output now?)))

Reply:
@AKalinich-Luxoft It's implemented by default inside `ArgumentParser`

Reply:
@ValeriiMalkov don't we need to prevent the script from running in this case? It should print out help and close